products/sle/main.pm

```diff
@@ -51,7 +51,7 @@ sub remove_desktop_needles {
 }
 
 sub is_server() {
-    return get_var('FLAVOR', '') =~ /^Server/;
+    return get_var('FLAVOR', '') =~ /^(Server|SAP)/;
```

Comment:
ok, I settled on leaving the flavor as it is. But please add a is_sles_for_sap function that checks the flavor and change is_server to return is_sles_for_sap || $match_on_flavor